src/dra/6D59C.c

```diff
@@ -294,7 +294,9 @@ void func_8010E0D0(s32 arg0) {
 }
 void func_8010E168(s32 arg0, s16 arg1) {
     if (arg0 == 0) {
-        CreateEntFactoryFromEntity(g_CurrentEntity, 0x15002C, 0);
+        // Create factory with unkA0 = 0x1500, blueprint #44.
+        // Blueprint 44 is to make child entity #11, or EntityPlayerBlinkWhite
+        CreateEntFactoryFromEntity(g_CurrentEntity, 0x1500 << 8 | 44, 0);
```

Comment:
An enum for these? Not sure what the best term is. `ENTITY_FACTORY_PLAYER_BLINK_WHITE`? This applies to the other usages

Reply:
I'm inclined to hold off until we can determine more of how the factory structure works, particularly that second argument, before starting to make enums and such. I think I'd like to keep decompiling things and writing descriptive comments of where things happen, and start figuring out what the different things are, before making enums, especially since that seems like a pretty long enum name which will end up making things spill onto extra lines.

Reply:
This is a nice idea, I will put something together in the next few days and update this PR :)